wagtail/tests/test_rich_text.py

```diff
@@ -131,6 +131,11 @@ def test_evaluate_value(self):
         value = RichText("<p>wagtail</p>")
         self.assertTrue(value)
 
+    def test_compare_value(self):
+        value1 = RichText("<p>wagtail</p>")
+        value2 = RichText("<p>wagtail</p>")
+        self.assertEquals(value1, value2)    
```

Comment:
Remove trailing whitespace and please use `assertEqual` instead of `assertEquals`.

```suggestion
        self.assertEqual(value1, value2)
```

Reply:
I'd suggest also adding some assertions for things that don't compare as equal, so that we know it's correctly returning False for those:

    value3 = RichText("<p>django</p>")
    self.assertNotEqual(value1, value3)
    self.assertNotEqual(value1, 12345)

Reply:
@NikilTn see above

Reply:
I've done the changes suggested by @gasman. can you please look into it @laymonage